tmt/steps/provision/mrack.py

```diff
@@ -275,6 +275,24 @@ def _transform_cpu_cores(
         children=[MrackHWBinOp('cores', beaker_operator, actual_value)])
 
 
+def _transform_cpu_vendor_name(
+        constraint: tmt.hardware.NumberConstraint,
```

Comment:
Shouldn't it be `TextConstraint` instead of `NumberConstraint`?

Reply:
Hi, thanks for review!  Yes you are right. Updated.